packages/common/log/src/platform/browser/index.ts

```diff
@@ -6,9 +6,21 @@ import { type LogOptions } from '../../config';
 
 // NOTE: Implementation for the browser. See `package.json`.
 export const loadOptions = (filepath?: string): LogOptions | undefined => {
+  let dxlog: string | undefined;
+  if (typeof localStorage === 'undefined') {
+    if ((globalThis as any).localStorage_dxlog) {
```

Comment:
a comment on when this gets used would be helpful. I was a little confused until I scrolled further down in the diff